tst/testinstall/opers/StructuralCopy.tst

```diff
@@ -0,0 +1,23 @@
+# Blist
+gap> a:=[true];; IsBlistRep(a);
```

Comment:
Add START_TEST / STOP_TEST to this .tst file, like in the other files in that dir?